src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Test/Index.cshtml

```diff
@@ -41,7 +46,7 @@
                             </div>
                         }
 
-                        if (Model.Discount != null && Model.IsAdvent)
+                        if (Model.Discount != null && isAdvent)
```

Comment:
as I mentioned review before - let's start with `isAdvent`

Reply:
You are commenting on outdated files